qcodes/dataset/data_set.py

```diff
@@ -540,6 +546,14 @@ def add_snapshot(self, snapshot: str, overwrite: bool=False) -> None:
             log.warning('This dataset already has a snapshot. Use overwrite'
                         '=True to overwrite that')
 
+    @property
+    def pristine(self) -> bool:
+        return not(self._started or self._completed)
+
+    @property
+    def running(self) -> bool:
```

Comment:
shall here be docstrings for these 3 properties?

Reply:
Ah, docstrings, yes, I've heard of those :)